src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -3204,8 +3204,8 @@ public DnsNameProperty(X509Certificate2 cert)
             // extract DNS name from subject distinguish name
             // if it exists and does not contain a comma
             // a comma, indicates it is not a DNS name
-            if (cert.Subject.StartsWith(distinguishedNamePrefix, System.StringComparison.InvariantCultureIgnoreCase) &&
-                cert.Subject.IndexOf(",", System.StringComparison.InvariantCulture) == -1)
+            if (cert.Subject.StartsWith(distinguishedNamePrefix, System.StringComparison.OrdinalIgnoreCase) &&
```

Comment:
Can we move this to another PR?  This looks like a bug fix

Reply:
Moved to #9987